include/ck/tensor_operation/gpu/device/device_normalization.hpp

```diff
@@ -46,13 +46,14 @@ template <typename XDataType,
           typename AccElementwiseOperation,
           index_t Rank,
           index_t NumReduceDim>
-struct DeviceNormalization2 : public BaseOperator
+struct DeviceLayernorm : public BaseOperator
```

Comment:
I inherit DeviceLayernorm from DeviceNormalization2,
After discussing with @qianfengz  , we thought it is not good.
But I thought adding DeviceNormalization2 -> DeviceLayernormBase and use it as external API is also not a good idea.
Hence, I rename these two classes as follow
DeviceLayernorm -> DeviceLayernormImpl
DeviceNormalization2 -> DeviceLayernorm
What do you think? @asroy

Reply:
It's ok for now. We can use current external interface and change them later if needed